components/Common/LanguageSelector/index.tsx

```diff
@@ -41,7 +41,7 @@ const LanguageSelector = () => {
   });
 
   return (
-    <div className={styles.container} ref={ref}>
+    <>
```

Comment:
I don't think this fragment is necessary anymore, right?

Reply:
It is, as there are two elements in the same level here